src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -660,11 +660,15 @@ private void removeTerritoriesWhereTransportsAreExposed() {
             ProBattleUtils.calculateBattleResults(player, t, patd.getUnits(), patd.getMaxEnemyDefenders(player, data),
                 patd.getBombardTerritoryMap().keySet(), true);
         int production = 0;
+        int isEnemyCapital = 0;
         final TerritoryAttachment ta = TerritoryAttachment.get(t);
         if (ta != null) {
           production = ta.getProduction();
+          if (ta.isCapital()) {
+            isEnemyCapital = 1;
```

Comment:
Correct. I'm planning to refactor all territory/factory value code into a separate class as its somewhat scattered right now. And yes it should consider the PUs it would capture rather than just adding a multiple to the territory's value.